### PR TITLE
Language endpoint adjustments

### PR DESCRIPTION
**Pull request by michau**
Some small adjustments for new form.

### CODE REVIEW

File: CRM/Speakcivi/Page/Lang.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 class CRM_Speakcivi_Page_Lang extends CRM_Speakcivi_Page_Post {
 
-  private $urlLanguageForm = '/form/preferred-language';
+  private $urlLanguageForm = '/language-switch';
```
Comment (scardinius) [inline CRM/Speakcivi/Page/Lang.php:7]: Looks fine. However should't be also "change" verb? finally to '/language-change' as above in subject "Language change from..."

Comment (michau) [reply CRM/Speakcivi/Page/Lang.php:7]: Fine - reworked the URLs for all translations to /language-change.

Comment (michau) [reply CRM/Speakcivi/Page/Lang.php]: in 93ce276a57b26163a8a9a30fc2b6f142c19365e3

**Comment by michau** [comment]
Can you merge and deploy?

**Pull request merged**